airbyte-workers/src/main/java/io/airbyte/workers/normalization/DefaultNormalizationRunner.java

```diff
@@ -163,6 +199,14 @@ public void close() throws Exception {
     }
   }
 
+  @Override
+  public Stream<AirbyteTraceMessage> getTraceMessages() {
+    if (airbyteMessagesByType != null) {
+      return airbyteMessagesByType.getOrDefault(Type.TRACE, new ArrayList<>()).stream().map(AirbyteMessage::getTrace);
+    }
+    return Stream.of();
```

Comment:
nit: `Stream.empty()` is more explicit about what we intent to return.

Reply:
👍